scss/_patterns_matrix.scss

```diff
@@ -6,38 +6,54 @@
     margin: 0;
     padding: 0;
 
+    @media (min-width: $breakpoint-small) {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
     &__item {
       @extend %clearfix;
-      border-bottom: 1px dotted $color-mid-dark;
-      margin-bottom: 1rem;
+      border-top: 1px dotted $color-mid-dark;
       padding: 1rem;
 
-      &:last-child {
-        border-bottom: 0;
+      &:empty {
+        display: none; // hide orphans
+      }
+
+
+      &:first-child {
+        border-top: 0;
       }
 
       @media (min-width: $breakpoint-small) { // 2 items per row
+        border-bottom: 1px dotted $color-mid-dark;
         border-right: 1px dotted $color-mid-dark;
-        display: table-cell;
-        float: left;
+        border-top: 0;
         margin-bottom: 0;
         padding-right: 1rem;
-        vertical-align: top;
         width: calc(50% - .5rem);
 
-        &:nth-child(2n) {
+        &:nth-child(even) {
           border-right: 0;
           margin-right: 0;
         }
 
-        &:nth-last-child(-n+2) {
+        &:nth-last-child(-n+3) {
```

Comment:
This line is causing the 3rd from end cell to have a missing border on medium screen.

![demo html](https://cloud.githubusercontent.com/assets/1413534/22109416/90edb998-de4f-11e6-99b8-c477781b36b2.png)